Create-Cluster.md

```diff
@@ -25,41 +25,59 @@ $ brew install terraform
 1. Create a single ["Machine to Machine"](https://auth0.com/docs/applications/machine-to-machine) Application. This application will allow terraform API access to create resources in Auth0. Select Management API with all scopes. Take note of the app's "Client ID & Secret" for use with terraform later.
   ![m2m app](auth0/tf.png)
 
-1. An **org-owned** [Github Oauth app](https://auth0.com/docs/connections/social/github), callback URL pointing to https://tenant-name.eu.auth0.com/login/callback
+1. Create An **org-owned** [Github Oauth app](https://auth0.com/docs/connections/social/github), callback URL pointing to https://tenant-name.eu.auth0.com/login/callback
```

Comment:
- Homepage URL =  https://login.apps.<cluster-name>.dsd.io

Note: If not admin for MoJ org, an admin will need to accept the 0Auth app.